src/test/java/org/openrewrite/kotlin/SuspendLambdaTest.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2023 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.kotlin;
+
+import org.junit.jupiter.api.Test;
+import org.openrewrite.java.ChangeType;
+import org.openrewrite.test.RecipeSpec;
+import org.openrewrite.test.RewriteTest;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.openrewrite.kotlin.Assertions.kotlin;
+
+public class SuspendLambdaTest implements RewriteTest {
```

Comment:
Hi @nomisRev , thanks for reporting the issue and creating a PR!
I'm happy with the test, but I think it should be added to [MethodDeclarationTest](https://github.com/openrewrite/rewrite-kotlin/blob/main/src/test/java/org/openrewrite/kotlin/tree/MethodDeclarationTest.java).

I'd prefer you to have credit for the changes -- would you please add the test the LambdaTest instead?

Info:
In the test package under `org.openrewrite.kotlin.tree` each of the tests use `ParserAssertions` instead of `Assertions`. `ParserAssertions` includes an additional step to check if the source is fully parsed by checking if any text exists in `Space`. All of the parsing tests exist there, and we apply similar conventions across other languages.

Changes:
- Added issue annotation with a link to the open issue.
- Changed test name to communicate what is parsed.

```java
    @Issue("https://github.com/openrewrite/rewrite-kotlin/issues/56")
    @Test
    void lambdaMethodParameterWithModifier() {
        rewriteRun(
          kotlin(
            """
            suspend fun example(
              title: String,
              verifyUnique: suspend (String) -> Boolean
            ): String = TODO()
            """,
            """            
            suspend fun example(
              title: String,
              verifyUnique: suspend (String) -> Boolean
            ): String = TODO()
            """
          )
        );
    }
```